### DISCUSSION TITLE
Annoying overlay while making screenshot

### DISCUSSION THREAD

**Discussion by martinkazv** [Eugeny/tabby#6120] [Q&A] [answered]
Hello, I have macOS 12.3 and i have newest version of Tabby.
Everytime I want to make a screenshot of the terminal, an overlay appears with the name of the current tab viewed and it is obstructing the view. I was unable to find a setting in Tabby to disable this behavior. Can you please help me on how to disable this, so the overlay does not appear? Thank yu very much!
![image](https://user-images.githubusercontent.com/37880981/162405101-3464bcb9-c279-4891-a6d9-60f977d6a4b4.png)

**Comment by Eugeny** [comment, marked as answer]
You can change the shortcut from Cmd-Shift to something else under Settings -> Hotkeys, or click the desktop before taking the screenshot

**Reply by martinkazv** [reply]
Thank you, as you said, the cmd-shift was used for this hotkey and I have disabled it.
![image](https://user-images.githubusercontent.com/37880981/162407981-d2634377-c7c2-44e9-b6d8-0f496e5ae88e.png)
Thank you

**Reply by lawweiliang** [reply]
Hi @martinkazv @Eugeny  ,

I would rather not remove the default key for Cmd-shift/Ctrl-shift (Show pane labels)
![image](https://user-images.githubusercontent.com/34936929/228132028-95539c79-0948-4f37-9e52-ee0f0d37c19a.png)

For print screen, I am using prnScr [^1]. 
It works for both Windows and Mac. The beauty of this print screen software is it allows you to upload the image to their server and share the link with your colleague.
Also, to print screen, you just need to click `print screen button` over your keyboard.

---
[^1]: https://prnt.sc/

**Comment by lawweiliang** [comment]
@martinkazv 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)

**Comment by benjaminv** [comment]
I still find this is unresolved. In my case, I am using a feature like screenshot active window. It relies on detection of current window to capture it. Tabby will add the overlay and dim the content largely no matter what shortcut is used. This is a little annoying rather than being a useful feature.
